docs/modules/operation/pages/provisioning/detectors/Win32ServiceDetector.adoc

```diff
@@ -0,0 +1,33 @@
+= Win32 Service Detector
+
+The Win32 Service Detector can be used for detecting the availability of services on Windows devices with SNMP enabled. It will make a query against the Microsoft LAN Manager MIB `.1.3.6.1.4.1.77` over SNMP to confirm the service.
+
+== Detector facts
+
+[options="autowidth"]
+|===
+| Implementation | `org.opennms.netmgt.provision.detector.snmp.Win32ServiceDetector`
+|===
+
+== Configuration and Use
+
+.Parameters for the Win32 Service Detector
+[options="header, autowidth"]
+|===
+| Parameter       | Description                                                                              | Required | Default value
+| `serviceName`   | The windows service name to query for. This should be the `Display name` field value for the Windows service in question as displayed in the `services.msc` applet.                                                 | Required | `Win32Service`
+|===
+
+== Example Configuration
+
+[source,xml]
+----
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<foreign-source date-stamp="2010-06-29T13:15:30.494+02:00" name="test" xmlns="http://xmlns.opennms.org/xsd/config/foreign-source">
+  <scan-interval>1d</scan-interval>
+      <detector name="WSUS" class="org.opennms.netmgt.provision.detector.snmp.Win32ServiceDetector">
+         <parameter key="serviceName" value="WSUS Service"/>
```

Comment:
Any hints we have to give about service names in Windows servers with another language and special characters in service names?

Reply:
Good question. We have this hint for the poller:
![image](https://user-images.githubusercontent.com/4995222/111803693-94bdb300-88cf-11eb-9335-59290e842e9b.png)

Not sure if it also would work for the detector.

Reply:
added some more details above which should help to solve this problem.

Reply:
Is the XML example required? We can configure via the user interface and the other detectors don't provide the XML definition either?